src/components/main/flight/FlightInfoScreen.js

```diff
@@ -239,265 +353,355 @@ export default class FlightInfoScreen extends React.Component {
+      );
     }
   }
 }
 
-const { width, height } = Dimensions.get('screen');
-
 const styles = StyleSheet.create({
-  containerLoading: {
+  background_image: {
     flex: 1,
-    flexDirection: 'column',
-    justifyContent: 'center',
-    height: height,
-    width: width,
-    backgroundColor: COLORS.white,
-  },
-  loadingIcon: {
-    color: COLORS.lightblue,
-    fontSize: 120,
-    textAlign: 'center'
+    backgroundColor: COLORS.black,
   },
-  loadingText: {
-    marginTop: '5%',
-    fontFamily: 'Montserrat-bold',
-    fontSize: 12,
-    color: COLORS.lightblue,
-    textAlign: 'center'
+  backDrop: {
+    flex: 1,
+    backgroundColor: "transparent",
   },
-  container: {
+  // Content Wrapper
+  innerView: {
     flex: 1,
-    flexDirection: 'column',
-    backgroundColor: COLORS.white,
-    height: height,
-    width: width
+    flexDirection: "column",
+    marginLeft: Math.round(moderateScale(105, 0.625)),
+    marginRight: Math.round(moderateScale(20, 0.0625)),
+    marginBottom: Math.round(moderateScale(30, 0.0625)),
+    marginTop: Math.round(moderateScale(80, 0.0625)),
+  },
+
+  // Top Inner View
+  topInnerView: {
+    flexDirection: "column",
+    marginBottom: Math.round(verticalScale(30)),
+    paddingBottom: Math.round(verticalScale(10)),
   },
-  containerTop: {
-    paddingLeft: width * 0.05,
-    paddingRight: width * 0.05,
-    paddingTop: height * 0.06,
-    marginBottom: height * 0.10,
-    backgroundColor: COLORS.white,
+  flightView: {
+    marginBottom: Math.round(verticalScale(20)),
   },
-  buttonBarNav: {
-    flexDirection: 'row',
-    height: height * 0.05,
-    justifyContent: 'space-between',
-    marginBottom: height * 0.05,
+  flightNumberLabel: {
+    color: COLORS.lightSandy,
+    fontFamily: "Poppins",
+    fontSize: Math.round(moderateScale(16, 0.0625)),
   },
-  buttonBarTop: {
-    flexDirection: 'row',
-    height: height * 0.04,
-    justifyContent: 'center',
-    width: '66%',
+  flightNumberText: {
+    color: COLORS.lightSandy,
+    fontFamily: "Poppins-bold",
+    fontSize: Math.round(moderateScale(40, 0.0625)),
   },
-  buttonBarBottom: {
-    flexDirection: 'row',
-    height: height * 0.04,
-    justifyContent: 'center',
+  flightInfoText: {
+    color: COLORS.lightSandy,
+    fontFamily: "Poppins",
+    fontSize: Math.round(moderateScale(16, 0.0625)),
   },
-  leftGreenButton: {
-    borderTopLeftRadius: 5,
-    borderBottomLeftRadius: 5,
-    marginRight: 3,
-    backgroundColor: COLORS.lightgreen,
+
+  // itinerary View
+  itineraryView: {
+    flexDirection: "row",
+    alignItems: "center",
+  },
+  plane_icon_view: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
   },
-  middleGreenButton: {
-    backgroundColor: COLORS.lightgreen,
-    marginRight: 3,
+  plane_icon: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
+    color: COLORS.lightSandy,
+    textAlign: "center",
+    fontSize: Math.round(moderateScale(35, 0.0625)),
```

Comment:
please add in `marginBottom: Math.round(moderateScale(10, 0.0625)),` for planeIcon

Reply:
My bad, you wanted this margin added. I am questioning why add in a margin to something that is at the same line height as all of the text in that row?

Reply:
It only slightly raises it a bit. on android it's not aligned as well as it could be. 
![image](https://user-images.githubusercontent.com/5898380/88121983-34c60f80-cb7c-11ea-99df-9fd274bbbc69.png)

Reply:
The Icon is smaller in font size. I think this is optically an illusion. try and set the background colors and see if this is the code or not. This may break things on IOS.

Reply:
This one can be left alone if it does break on ios.